lib/metadata.js

```diff
@@ -37,9 +36,8 @@ function Metadata(url, authtype, options) {
   if (!url) {
     throw new Error('Metadata: url is a required argument');
   }
-  if (!authtype) {
-    throw new Error(`OIDCBearerStrategy requires an authentication type specified to metadata parser.
-      Valid types are "saml", "wsfed", or "odic"`);
+  if (!authtype || authtype !== 'oidc') {
```

Comment:
Does it mean that we do not want to support wsfedstrategy and saml  ? Looks like Metadata is used by them -  they call it with "saml" and "wsfed"  authtypes, do we want to remove wsfedstrategy and saml  completely ?